app/retail/templates/shared/footer_scripts_lite.html

```diff
@@ -29,8 +29,8 @@
   <script src="https://cdn.jsdelivr.net/npm/vue/dist/vue.js"></script>
 {% endif %}
 {% include 'shared/sentry.html' %}
-<script src="{% static "v2/js/lib/popper.min.js" %}"></script>
-<script src="{% static "v2/js/lib/bootstrap.min.js" %}" crossorigin="anonymous"></script>
```

Comment:
The idea is to move bootstrap.js to footer_scripts and footer_scripts_lite since now is used in most parts and is part of the framework. 
I had changed that in this PR https://github.com/gitcoinco/web/pull/4727

Reply:
got it - maybe we should just have a seperate `compress` section here..

Reply:
yes for sure we can add it later. In some point we will get in a place we can just use the "lite" version and deprecate the other, but need to check since there are diff parts of the site with diff needs and for example the shared.js is something Im trying to avoid to be loaded in all pages. Maybe having one compress for  footer_scripts_lite and one for footer_scripts